### DISCUSSION TITLE
Dynamic name, and descriptons

### DISCUSSION THREAD

**Discussion by jxnl** [instructor-ai/instructor-js#53] [Ideas]
In the Python implementation, instead of using `name: "response_model"`, we utilize variable names to assist the model. Additionally, we include the description `Correctly Extracted {response_model.name}` to enhance the effectiveness of prompts. This could be the reason for the frequent failures in our tests compared to Python.

cc: @roodboi 

should be add a name: attribute to the create call? can we detect if theres no description and add one?

**Comment by lukasb** [comment]
Has this been implemented?

**Reply by jxnl** [reply]
you need to set name {"response_model": ...,  "name": ...}

**Reply by roodboi** [reply]
that should be the case currently - response_model: { name, schema } 

https://github.com/instructor-ai/instructor-js/blob/main/examples/extract_user_stream/index.ts#L59
